crates/bitwarden-uniffi/src/lib.rs

```diff
@@ -88,6 +88,23 @@ impl Client {
 
         Ok(res.text().await.map_err(bitwarden_core::Error::Reqwest)?)
     }
+
+    pub fn generate_ssh_key(
+        &self,
+        key_algorithm: bitwarden_ssh::generator::KeyAlgorithm,
+    ) -> Result<bitwarden_ssh::SshKey> {
+        bitwarden_ssh::generator::generate_sshkey(key_algorithm)
+            .map_err(|e| error::BitwardenError::E(error::Error::SshGeneration(e)))
+    }
+
+    pub fn import_ssh_key(
+        &self,
+        imported_key: String,
+        password: Option<String>,
+    ) -> Result<bitwarden_ssh::SshKey> {
+        bitwarden_ssh::import::import_key(imported_key, password)
+            .map_err(|e| error::BitwardenError::E(error::Error::SshImport(e)))
+    }
```

Comment:
**issue:** Please make a client for these, we don't want them in the root.

Reply:
Updated and moved to a client, should we do the same for wasm in a follow-up pr?